Add the from field to the example document. (#702)
firebase/extensions: firestore-send-email/README.md

```js
admin.firestore().collection('mail').add({
  to: 'someone@example.com',
  from: 'somebodyelse@example.com',
  message: {
    subject: 'Hello from Firebase!',
    html: 'This is an <code>HTML</code> email body.',
  },
})
```

> **custom senders**
> 
> The `from` field is an optional parameter. If a custom sender is included, please ensure the receiving SMTP server accepts this fields. Typically, a provider will use the email provided by the configured account details if a custom sender is not allowed.
> 

Reviewer: ```suggestion
**Custom senders**

The `from` field is an optional parameter. If you specify a value for this field, ensure the receiving SMTP server accepts custom senders in this field. Typically, a provider will use the email address provided by the configured account if the provider doesn't allow custom senders.

```
